help_to_heat/frontdoor/views.py

```diff
@@ -595,6 +595,19 @@ def handle_post(self, request, session_id, page_name, data, is_change_page):
             return redirect("frontdoor:page", session_id=session_id, page_name="property-type")
         return super().handle_post(request, session_id, page_name, data, is_change_page)
 
+    def get_prev_next_urls(self, session_id, page_name):
+        session_data = interface.api.session.get_session(session_id)
+        park_home = session_data.get("park_home")
+        country = session_data.get("country")
+        epc_rating = session_data.get("epc_rating", "Not found")
+
+        if park_home == "Yes" and (country == "Scotland" or epc_rating == "Not found"):
+            _, next_page_url = get_prev_next_urls(session_id, page_name)
```

Comment:
Is there a subtle difference between this and `super().get_prev_next_urls` below?

Reply:
I don't believe there's a functional difference at present, as super().get_prev_next_urls immediately passes the inputs through to get_prev_next_urls, and returns the result of that function without any changes.

I'm reusing the style established for special cases within the method on line 709.